src/operator-manual/opentelemetry/01-quickstart.md

```diff
@@ -0,0 +1,102 @@
+# OpenTelemetry
+
+[OpenTelemetry](https://opentelemetry.io/) is a Cloud Native Computing Foundation framework for
+observability. It enables your microservices to provide metrics, logs and traces.
+
+Kubewarden's `policy-server` is instrumented with the OpenTelemetry SDK, reporting data to an
+OpenTelemetry collector -- called the agent.
+
+In addition to the OpenTelemetry collector running as an agent in a sidecar, it can also be deployed
+on the cluster as a gateway. In this operational model, the OpenTelemetry collector running as an
+agent reports metrics, logs and traces to another OpenTelemetry collector running as a gateway,
+configured to report this data to third party services, either running inside of the cluster or
+outside.
```

Comment:
```suggestion
The OpenTelemetry collector can be either run as a sidecar agent, or it can be deployed
inside of the cluster as a gateway.

This documentation focuses on deploying the OpenTelemetry agent as a container sidecar. For more information about
the other deployment modes, please refer to OpenTelemetry official documentation.
```

Reply:
Here I'm trying to put into words that it's possible to:

1. Component -> Agent -> 3rd party services (e.g. datadog). This requires to configure every sidecar with third party credentials.
2. Component -> Gateway -> 3rd party services. This requires to reconfigure the gateway whenever a service needs to change something.
3. Component -> Agent -> Gateway -> 3rd party services: This is what I'm trying to mention here. In this case, the agent is configured to send everything through OTLP to another OpenTelemetry collector (the gateway), that is configured once to report data to third party services.

Reply:
I see, then I would revisit my suggestion to be something like:

> By following this documentation, we will integrate OpenTelemetry using the following architecture:
>   * Each Pod of the Kubewarden stack will have a OpenTelemetry sidecar
>   * The sidecar receives tracing and monitoring information from the Kubewarden component via the OpenTelemetry Protocol (OTLP)
>   * The OpenTelemetry collector will:
>     - Send the trace events to a central Jaeger instance
>     - Expose Prometheus metrics on port <XXXX>
>
>  For more information about the other deployment modes, please refer to OpenTelemetry official documentation.